src/unity/python/turicreate/CMakeLists.txt

```diff
@@ -73,4 +73,17 @@ elseif(LINUX)
     COMMAND ${CMAKE_COMMAND} -E copy_if_different "${CMAKE_SOURCE_DIR}/src/unity/python/turicreate/visualization/vg2png" "${CMAKE_CURRENT_BINARY_DIR}"
     COMMAND ${CMAKE_COMMAND} -E copy_if_different "${CMAKE_SOURCE_DIR}/src/unity/python/turicreate/visualization/vg2svg" "${CMAKE_CURRENT_BINARY_DIR}"
   )
+
+  if(CLANG)
+    find_library(LIBCXX c++)
+    find_library(LIBCXXABI c++abi)
+    find_library(LIBUNWIND unwind)
+    add_custom_target(
+      copy_libcxx_deps ALL
+      COMMAND cp ${LIBCXX}.1.0 ${CMAKE_CURRENT_BINARY_DIR}/libc++.so.1
```

Comment:
Is this harmless in the APPLE case?

Reply:
Probably not. That's why this is still a WIP PR: I want to try building on all our supported configurations currently and make sure it doesn't break anything. I think this probably does break on APPLE -- good catch!

Reply:
What are the holdups here?  How can we get this in?

Reply:
Ok, this line actually should be harmless in the APPLE case since it's nested in an `elseif(LINUX)`. I'll test locally to make sure.